test-kotest/build.gradle

```diff
@@ -0,0 +1,42 @@
+plugins {
+    id "org.jetbrains.kotlin.jvm" version "1.3.71"
+    id "org.jetbrains.kotlin.kapt" version "1.3.71"
+}
+
+ext {
+    micronautHibernateJpaVersion = "1.3.0"
+    micronautJdbcTomcatVersion = "1.3.0"
+    kotlinVersion = "1.3.71"
+}
+
+dependencies {
+    api project(":test-core")
+    implementation "io.micronaut:micronaut-inject:$micronautVersion"
+    api "io.micronaut:micronaut-runtime:$micronautVersion"    
+    api "io.kotest:kotest-runner-junit5-jvm:$kotestVersion"
+
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:$kotlinVersion"
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlinVersion"
+
+    testImplementation "io.mockk:mockk:1.9.3"
+    testImplementation "io.micronaut:micronaut-http-client:$micronautVersion"
+    testImplementation "io.micronaut:micronaut-http-server-netty:$micronautVersion"
+    kaptTest "io.micronaut:micronaut-inject-java:$micronautVersion"
+
+    testImplementation "io.micronaut.configuration:micronaut-hibernate-jpa:$micronautHibernateJpaVersion"
+    testRuntimeOnly "io.micronaut.configuration:micronaut-jdbc-tomcat:$micronautJdbcTomcatVersion"
+    testRuntimeOnly "com.h2database:h2:1.4.200"
+    
+    testImplementation "io.kotest:kotest-assertions-core-jvm:4.0.3"
```

Comment:
Should be depending on `$kotestVersion`as well, shouldn't it?

Reply:
Yep, missed this one